### PR TITLE
[FR] Add "osu!dev Discord server"

### PR DESCRIPTION
**Pull request by Ganondorfosu**
French translation of osu! wiki - osu!dev Discord server

### CODE REVIEW

File: wiki/osu!dev_Discord_server/fr.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+tags:
+  - development
+---
+
+# Serveur Discord osu!dev
+
+**osu!dev** est le serveur [Discord](https://discordapp.com "Discord's website") officiel pour travailler sur les projets open-source et communautaires d'osu!. C'est le lieu principal pour contribuer au développement d'osu! et entrer en contact avec l'[Equipe osu!](/wiki/People/The_Team).
```
Comment (Pachiru) [inline wiki/osu!dev_Discord_server/fr.md:8]:
```suggestion
**osu!dev** est le serveur [Discord](https://discordapp.com "site officiel de Discord") officiel pour travailler sur les projets open-source et communautaires relatifs à osu!. C'est le lieu principal pour contribuer au développement d'osu! et entrer en contact avec l'[équipe d'osu!](/wiki/People/The_Team).
```

File: wiki/osu!dev_Discord_server/fr.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Le lien d'invitation permanent est <https://discord.gg/ppy>.
+
+## Vérification
+
+Les nouveaux membres du serveur ne sont pas autorisés à envoyer des messages sur le serveur par défaut. S'ils veulent envoyer des messages, ils peuvent poster un message dans `#arrival` avec un bref résumé de leurs intentions. Un membre de l'équipe osu! ajoutera un rôle qui retirera cette restriction.
```
Comment (Pachiru) [inline wiki/osu!dev_Discord_server/fr.md:14]:
```suggestion
Par défaut, seul les utilisateurs vérifiés ont l'autorisation de discuter dans le reste du serveur Discord. Si un utilisateur non-vérifié souhaite le devenir, alors celui-ci doit poster un message dans le canal `#arrival` du serveur Discord accompagné de ses motivations et intentions. Une fois ceci fait, un membre de l'équipe d'osu! s'occupera de vérifier l'utilisateur. *Prenez note que la vérification n'est pas immédiate et qu'elle peut prendre un certain temps.* 
```

File: wiki/osu!dev_Discord_server/fr.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Projets
+
+Chaque développement principal sur le serveur a son propre canal de discussion:
```
Comment (Pachiru) [inline wiki/osu!dev_Discord_server/fr.md:18]:
```suggestion
Chaque partie du développement possède son propre canal de discussion:
```

File: wiki/osu!dev_Discord_server/fr.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+| Project | Channel |
```
Comment (Pachiru) [inline wiki/osu!dev_Discord_server/fr.md:20]:
```suggestion
| Projet | Canal |
```

File: wiki/osu!dev_Discord_server/fr.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+| :-- | :-- |
+| [osu!lazer](https://github.com/ppy/osu "ppy/osu sur GitHub") | `#osu-lazer` |
+| [osu!framework](https://github.com/ppy/osu-framework "ppy/osu-framework sur GitHub") | `#osu-framework` |
+| [osu!web](https://github.com/ppy/osu-web "ppy/osu-web sur GitHub") | `#osu-web` |
+| Le [star rating](/wiki/Beatmapping/Star_rating) et les [points de performance](/wiki/Performance_Points) dans le mode [osu!standard](/wiki/Game_mode/osu!)  | `#difficulty-osu` |
```
Comment (Pachiru) [inline wiki/osu!dev_Discord_server/fr.md:25]:
```suggestion
| Discussions et modifications apportées au [Star Rating](/wiki/Beatmapping/Star_rating) et aux [points de performance](/wiki/Performance_Points) dans le mode [osu!standard](/wiki/Game_mode/osu!)  | `#difficulty-osu` |
```

File: wiki/osu!dev_Discord_server/fr.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+| Le star rating et les points de performance dans le mode [osu!taiko](/wiki/Game_mode/osu!taiko) | `#difficulty-taiko` |
```
Comment (Pachiru) [inline wiki/osu!dev_Discord_server/fr.md:26]:
```suggestion
| Discussions et modifications apportées au Star Rating et aux points de performance dans le mode [osu!taiko](/wiki/Game_mode/osu!taiko) | `#difficulty-taiko` |
```

File: wiki/osu!dev_Discord_server/fr.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+| Le star rating et les points de performance dans le mode [osu!catch](/wiki/Game_mode/osu!catch) | `#difficulty-catch` |
```
Comment (Pachiru) [inline wiki/osu!dev_Discord_server/fr.md:27]:
```suggestion
| Discussions et modifications apportées au Star Rating et aux points de performance dans le mode [osu!catch](/wiki/Game_mode/osu!catch) | `#difficulty-catch` |
```

File: wiki/osu!dev_Discord_server/fr.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+| Le star rating et les points de performance dans le mode [osu!mania](/wiki/Game_mode/osu!mania) | `#difficulty-mania` |
```
Comment (Pachiru) [inline wiki/osu!dev_Discord_server/fr.md:28]:
```suggestion
| Discussions et modifications apportées au Star Rating et aux points de performance dans le mode [osu!mania](/wiki/Game_mode/osu!mania) | `#difficulty-mania` |
```

File: wiki/osu!dev_Discord_server/fr.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+| [Beatmap Spotlights](/wiki/Beatmap_Spotlights) | `#osu-spotlights` |
+| [osu! wiki](https://github.com/ppy/osu-wiki "ppy/osu-wiki sur GitHub") | `#osu-wiki` |
+| [Project Loved](/wiki/Project_Loved) | `#osu-loved` |
+| Discussions à propos du [Modding](/wiki/Modding) et les [NAT meetings](/wiki/Modding/NAT_meetings) | `#modding` |
+| [Mappers' Guild](/wiki/Mappers_Guild) | `#mappers-guild` |
```
Comment (Pachiru) [inline wiki/osu!dev_Discord_server/fr.md:33]:
```suggestion
| Discussions autour de la [Mappers' Guild](/wiki/Mappers_Guild) ainsi que les mises à jour du statut des beatmaps et de l'ajout de quêtes liées à ce projet. | `#mappers-guild` |
```

**Review by Pachiru — CHANGES_REQUESTED** [review]
Update this and it should be good to go

File: wiki/osu!dev_Discord_server/fr.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+Le lien d'invitation permanent est <https://discord.gg/ppy>.
+
+## Vérification
+
+Par défaut, seul les utilisateurs vérifiés ont l'autorisation de discuter dans le reste du serveur Discord. Si un utilisateur non-vérifié souhaite le devenir, alors celui-ci doit poster un message dans le canal `#arrival` du serveur Discord accompagné de ses motivations et intentions. Une fois ceci fait, un membre de l'équipe d'osu! s'occupera de vérifier l'utilisateur. *Prenez note que la vérification n'est pas immédiate et qu'elle peut prendre un certain temps.* 
+
+## Projets
+
+Chaque partie du développement possède son propre canal de discussion:
```
Comment (DoryanR) [inline wiki/osu!dev_Discord_server/fr.md:18]:
```suggestion
Chaque partie du développement possède son propre canal de discussion :
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
After modifications, it will be good for me.

File: wiki/osu!dev_Discord_server/fr.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
+---
+tags:
+  - development
+---
+
+# Serveur Discord osu!dev
+
+**osu!dev** est le serveur [Discord](https://discordapp.com "site officiel de Discord") officiel pour travailler sur les projets open-source et communautaires relatifs à osu!. C'est le lieu principal pour contribuer au développement d'osu! et entrer en contact avec l'[équipe d'osu!](/wiki/People/The_Team).
```
Comment (DoryanR) [inline wiki/osu!dev_Discord_server/fr.md:8]:
```suggestion
**osu!dev** est le serveur [Discord](https://discordapp.com "site officiel de Discord") officiel pour travailler sur les projets open-source et communautaires relatifs à osu!. C'est le lieu principal pour contribuer au développement d'osu! et entrer en contact avec [le staff d'osu!](/wiki/People/The_Team).
```

File: wiki/osu!dev_Discord_server/fr.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+| [osu!web](https://github.com/ppy/osu-web "ppy/osu-web sur GitHub") | `#osu-web` |
+| Discussions et modifications apportées au [Star Rating](/wiki/Beatmapping/Star_rating) et aux [points de performance](/wiki/Performance_Points) dans le mode [osu!standard](/wiki/Game_mode/osu!)  | `#difficulty-osu` |
+| Discussions et modifications apportées au Star Rating et aux points de performance dans le mode [osu!taiko](/wiki/Game_mode/osu!taiko) | `#difficulty-taiko` |
+| Discussions et modifications apportées au Star Rating et aux points de performance dans le mode [osu!catch](/wiki/Game_mode/osu!catch) | `#difficulty-catch` |
+| Discussions et modifications apportées au Star Rating et aux points de performance dans le mode [osu!mania](/wiki/Game_mode/osu!mania) | `#difficulty-mania` |
+| [Beatmap Spotlights](/wiki/Beatmap_Spotlights) | `#osu-spotlights` |
+| [osu! wiki](https://github.com/ppy/osu-wiki "ppy/osu-wiki sur GitHub") | `#osu-wiki` |
+| [Project Loved](/wiki/Project_Loved) | `#osu-loved` |
+| Discussions à propos du [Modding](/wiki/Modding) et les [NAT meetings](/wiki/Modding/NAT_meetings) | `#modding` |
```
Comment (DoryanR) [inline wiki/osu!dev_Discord_server/fr.md:32]:
```suggestion
| Discussions à propos du [Modding](/wiki/Modding) et des [NAT meetings](/wiki/Modding/NAT_meetings) | `#modding` |
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
Still some things to work on.

File: wiki/osu!dev_Discord_server/fr.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+| [osu!web](https://github.com/ppy/osu-web "ppy/osu-web sur GitHub") | `#osu-web` |
+| Discussions et modifications apportées au [Star Rating](/wiki/Beatmapping/Star_rating) et aux [points de performance](/wiki/Performance_Points) dans le mode [osu!standard](/wiki/Game_mode/osu!)  | `#difficulty-osu` |
+| Discussions et modifications apportées au Star Rating et aux points de performance dans le mode [osu!taiko](/wiki/Game_mode/osu!taiko) | `#difficulty-taiko` |
+| Discussions et modifications apportées au Star Rating et aux points de performance dans le mode [osu!catch](/wiki/Game_mode/osu!catch) | `#difficulty-catch` |
+| Discussions et modifications apportées au Star Rating et aux points de performance dans le mode [osu!mania](/wiki/Game_mode/osu!mania) | `#difficulty-mania` |
+| [Beatmap Spotlights](/wiki/Beatmap_Spotlights) | `#osu-spotlights` |
+| [osu! wiki](https://github.com/ppy/osu-wiki "ppy/osu-wiki sur GitHub") | `#osu-wiki` |
+| [Project Loved](/wiki/Project_Loved) | `#osu-loved` |
+| Discussions sur le [Modding](/wiki/Modding) et [réunions de la NAT](/wiki/Modding/NAT_meetings) | `#modding` |
```
Comment (DoryanR) [inline wiki/osu!dev_Discord_server/fr.md:32]:
```suggestion
| Discussions sur le [Modding](/wiki/Modding) et les [réunions de la NAT](/wiki/Modding/NAT_meetings) | `#modding` |
```

**Comment by DoryanR** [comment]
@Ganondorfosu Need you to apply the proposed changes

File: wiki/osu!dev_Discord_server/fr.md [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+## Projets
+
+Chaque partie du développement possède son propre canal de discussion :
+
+| Projet | Canal |
+| :-- | :-- |
+| [osu!lazer](https://github.com/ppy/osu "ppy/osu sur GitHub") | `#osu-lazer` |
+| [osu!framework](https://github.com/ppy/osu-framework "ppy/osu-framework sur GitHub") | `#osu-framework` |
+| [osu!web](https://github.com/ppy/osu-web "ppy/osu-web sur GitHub") | `#osu-web` |
+| Discussions et modifications apportées au [Star Rating](/wiki/Beatmapping/Star_rating) et aux [points de performance](/wiki/Performance_Points) dans le mode [osu!standard](/wiki/Game_mode/osu!)  | `#difficulty-osu` |
```
Comment (DoryanR) [inline wiki/osu!dev_Discord_server/fr.md:25]:
```suggestion
| Discussions et modifications apportées au [Star Rating](/wiki/Beatmapping/Star_rating) et aux [points de performance](/wiki/Performance_Points) dans le mode [osu!](/wiki/Game_mode/osu!)  | `#difficulty-osu` |
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

File: wiki/osu!dev_Discord_server/fr.md [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+
+# Serveur Discord osu!dev
+
+**osu!dev** est le serveur [Discord](https://discordapp.com "site officiel de Discord") officiel pour travailler sur les projets open-source et communautaires relatifs à osu!. C'est le lieu principal pour contribuer au développement d'osu! et entrer en contact avec [le staff d'osu!](/wiki/People/The_Team).
+
+Le lien d'invitation permanent est <https://discord.gg/ppy>.
+
+## Vérification
+
+Par défaut, seul les utilisateurs vérifiés ont l'autorisation de discuter dans le reste du serveur Discord. Si un utilisateur non-vérifié souhaite le devenir, alors celui-ci doit poster un message dans le canal `#arrival` du serveur Discord accompagné de ses motivations et intentions. Une fois ceci fait, un membre de l'équipe d'osu! s'occupera de vérifier l'utilisateur. *Prenez note que la vérification n'est pas immédiate et qu'elle peut prendre un certain temps.*
```
Comment (osuShiro) [inline wiki/osu!dev_Discord_server/fr.md:14]:
```suggestion
Par défaut, seuls les utilisateurs vérifiés ont l'autorisation de discuter dans le reste du serveur Discord. Si un utilisateur non-vérifié souhaite le devenir, celui-ci doit poster un message dans le canal `#arrival` du serveur Discord accompagné de ses motivations et intentions. Une fois ceci fait, un membre de l'équipe d'osu! s'occupera de vérifier l'utilisateur. *Prenez note que la vérification n'est pas immédiate et qu'elle peut prendre un certain temps.* 
```

**Review by osuShiro — DISMISSED** [review]

File: wiki/osu!dev_Discord_server/fr.md [hunk 16]
```diff
@@ -0,0 +1,33 @@
+---
+tags:
+  - development
```
Comment (DoryanR) [inline wiki/osu!dev_Discord_server/fr.md:3]:
```suggestion
  - développement
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

**Review by DoryanR — DISMISSED** [review]

File: wiki/osu!dev_Discord_server/fr.md [hunk 17]
```diff
@@ -0,0 +1,33 @@
+---
+tags:
+  - développement
```
Comment (TicClick) [inline wiki/osu!dev_Discord_server/fr.md:3]: please add tags from the original article

File: wiki/osu!dev_Discord_server/fr.md [hunk 18]
```diff
@@ -0,0 +1,33 @@
...
+| Discussions et modifications apportées au Star Rating et aux points de performance dans le mode [osu!taiko](/wiki/Game_mode/osu!taiko) | `#difficulty-taiko` |
+| Discussions et modifications apportées au Star Rating et aux points de performance dans le mode [osu!catch](/wiki/Game_mode/osu!catch) | `#difficulty-catch` |
+| Discussions et modifications apportées au Star Rating et aux points de performance dans le mode [osu!mania](/wiki/Game_mode/osu!mania) | `#difficulty-mania` |
+| [Beatmap Spotlights](/wiki/Beatmap_Spotlights) | `#osu-spotlights` |
+| [osu! wiki](https://github.com/ppy/osu-wiki "ppy/osu-wiki sur GitHub") | `#osu-wiki` |
+| [Project Loved](/wiki/Project_Loved) | `#osu-loved` |
+| Discussions sur le [Modding](/wiki/Modding)/[Réunions de la NAT](/wiki/Modding/NAT_meetings) | `#modding` |
+| Discussions autour de la [Mappers' Guild](/wiki/Mappers_Guild) ainsi que les mises à jour du statut des beatmaps et de l'ajout de quêtes liées à ce projet. | `#mappers-guild` |
```
Comment (TicClick) [inline wiki/osu!dev_Discord_server/fr.md:33]: this is missing from the original page, mind removing it for now?

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/osu!dev_Discord_server/fr.md (hunk 17 above)
Comment (Ganondorfosu) [reply wiki/osu!dev_Discord_server/fr.md:3]: I don't see any other tag on the original article, am I missing something ?

Comment (TicClick) [reply wiki/osu!dev_Discord_server/fr.md:3]: `development` it is, as opposed to `développement` -- you are free to keep either the original one, or both if you want this article to also rank higher in search results for `development` and `développement`

Comment (TicClick) [inline wiki/osu!dev_Discord_server/fr.md:3]:
```suggestion
  - development
  - développement
```

**Review by TicClick — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]
ok ci failures are [unrelated](https://www.githubstatus.com/incidents/m16jzl31gnqt) to the recent changes, and I haven't seen any before these, so let's push this

**Pull request merged**